src/main/java/com/github/fishio/achievements/AchievementIO.java

```diff
@@ -0,0 +1,91 @@
+package com.github.fishio.achievements;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashMap;
+
+import com.esotericsoftware.yamlbeans.YamlReader;
+import com.esotericsoftware.yamlbeans.YamlWriter;
+import com.github.fishio.logging.Log;
+import com.github.fishio.logging.LogLevel;
+
+/**
+ * This class is responsible for the Input and Output of Achievements to and
+ * from file, making sure that the achievements are saved and loaded.
+ *
+ */
+public class AchievementIO {
```

Comment:
*Auto-generated comment*

    Utility classes should not have a public or default constructor.